features.md

```diff
@@ -160,3 +160,78 @@ Known limitations
 > editors, e.g. VSCode, provide a way to preview the rename changes before
 > applying them; while some just apply the changes directly.
 {:.tip}
+
+## Semantic Highlighting
+
+clangd computes semantic highlight tokens to enhance the source code theming. {:.v9}
```

Comment:
I did check the website rendering locally, and the `{:.v9}` is still not rendered.

It looks like it needs to go on its own line.

Otherwise, looks good!

Reply:
I went ahead and made this change myself :)